scripts/image

```diff
@@ -138,8 +138,13 @@ echo -e "PRETTY_NAME=\"$DISTRONAME ($LIBREELEC_BUILD) - Version: $LIBREELEC_VERS
 echo -e "HOME_URL=\"https://libreelec.tv\"" >> $INSTALL/etc/os-release
 echo -e "BUG_REPORT_URL=\"$ORIGIN_URL\"" >> $INSTALL/etc/os-release
 echo -e "BUILD_ID=\"$GIT_HASH\"" >> $INSTALL/etc/os-release
-echo -e "OPENELEC_ARCH=\"$PROJECT.$TARGET_ARCH\"" >> $INSTALL/etc/os-release
-echo -e "LIBREELEC_ARCH=\"$PROJECT.$TARGET_ARCH\"" >> $INSTALL/etc/os-release
+if [ -z "$DEVICE" ]; then
+  echo -e "OPENELEC_ARCH=\"$PROJECT.$TARGET_ARCH\"" >> $INSTALL/etc/os-release
+  echo -e "LIBREELEC_ARCH=\"$PROJECT.$TARGET_ARCH\"" >> $INSTALL/etc/os-release
+else
+  echo -e "OPENELEC_ARCH=\"$DEVICE.$TARGET_ARCH\"" >> $INSTALL/etc/os-release
```

Comment:
This assumes DEVICE is somewhat unique, and you can't have the same device name for two different projects (particularly if they're the same TARGET_ARCH).

Wouldn't `$PROJECT.$DEVICE.$TARGET_ARCH` be a more accurate reflection of the build? I realise this may require back end changes, while the change above will work with existing infrastructure at the expense of losing some information (ie. PROJECT).

See my comments in #1357.